<launch>
    <arg name="node_name" default="imu"/>
    <arg name="topic_name" default="imu_sensor"/>
    <arg name="topic_buffer_size" default="10"/>
    <arg name="device_id" default="imu"/>
    <arg name="publish_rate" default="30"/>
    <arg name="accel_filter_loc" default="$(find peripherals)/csv/lowpass.csv"/>
    <arg name="vel_filter_loc" default="$(find peripherals)/csv/bandstop_tested.csv"/>

    <node name="$(arg node_name)" pkg="peripherals" type="imu" respawn="true" respawn_delay="1" >
        <param name="topic_name" value="$(arg topic_name)"/>
        <param name="topic_buffer_size" value="$(arg topic_buffer_size)"/>
        <param name="device_id" value="$(arg device_id)"/>
        <param name="publish_rate" value="$(arg publish_rate)"/>
        <param name="accel_filter_loc" value="$(arg accel_filter_loc)"/>
        <param name="vel_filter_loc" value="$(arg vel_filter_loc)"/>
    </node>
</launch>
